src/main/java/gregtech/api/block/machines/BlockMachine.java

```diff
@@ -84,7 +84,6 @@ public BlockMachine() {
         setHardness(6.0f);
         setResistance(6.0f);
         setTranslationKey("unnamed");
-        setHarvestLevel("wrench", 1);
```

Comment:
There are two main reasons why I removed this line.
1. This line was overriding all other attempts to actually set the harvest level of the clipboard, and since I didn't want someone to accidentally break it by trying to click a checkbox, I removed it.
2. Fortunately, this did not actually affect the mining level nor harvest tool for most MTEs. I tested this in-game, and this didn't allow me to break machines with my fist.
So, unless my testing wasn't very good here, I think this should be fine.